packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -60,10 +60,15 @@ function UncontrolledInnerBlocks( props ) {
 		orientation,
 		placeholder,
 		layout,
+		name,
+		innerBlocks,
+		parentLock,
+		context,
 	} = props;
 
 	useNestedSettingsUpdate(
 		clientId,
+		parentLock,
```

Comment:
Note that all of these are internal sub hooks for only this component, so it's ok to change them.